atlp-rwanda/code-ninjas-fe
# #181414720 User profile settings

### What does this PR do?

Implement user profile settings

### Description of Task to be completed?
After user registration on the app, he/she have the capability to get his/her profile, complete it with the remaining data and be able to update it later on.
### How should this be manually tested?
Get started By Running commands bellow:
  - clone repo
  - switch to the branch: [ft-user-profile-181414720]([ft-user-profile-181414720](https://github.com/atlp-rwanda/code-ninjas-fe/tree/ft-user-profile-181414720))
  - Run `yarn install`: to install all dependencies
  - Run `yarn dev`: to start the application
  - Open your browser & and login to the app.
  - After logging in, navigate to the profile page on the dashboard.
  - From there, you should be able to see your profile and the filled data in their respective fields.

### Routes:

`
<host>/dashboard/profile
`

### Any background context you want to provide?
N/A
### What are the relevant pivotal tracker stories?
[#181414720](https://www.pivotaltracker.com/story/show/181414720)
### Screenshots (if appropriate)
<img width="1439" alt="Screen Shot 2022-06-14 at 13 05 52" src="https://user-images.githubusercontent.com/75941936/173563353-aec94283-ff85-47a4-ad82-219a62bbf9c6.png">

### Questions:
N/A
